### PR TITLE
Changed REST routes

### PR DESCRIPTION
**Pull request by andys8**
I saw the example REST API has uncommon routes: `POST /user/add` and `GET /user/get/<name>`. The information that a new user entity is created or should be retrieved is encoded in the used HTTP method. Therefore I'd suggest removing it from the path because of redundancy.

The PR changes it to the expected interface:
* `POST /user`
* `GET /user/<name>`

Happy to hear your feadback and if this change is welcome 😃

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,14 +5,14 @@ database.
 
 You can build and run the project with [stack](http://haskellstack.org/), e.g.:
 
-``` bash
+```shell
```
Comment (soenkehahn) [inline README.md]: Why did you change this from `bash` to `shell`?

Comment (andys8) [reply README.md]: [Github Help](https://help.github.com/articles/creating-and-highlighting-code-blocks/) points to this [definition](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L4187-L4195) of possible languages in markdown code blocks. `shell`, `bash`, `zsh` and others are aliases. The change does effectively nothing. I'd prefer the keyword `shell` because it's agnostic and the named commands work in any shell, not only the bash shell. But all in all this is absolutely a minor change.

**Comment by soenkehahn** [comment]
LGTM, thanks a lot. I've invited you to be a collaborator on this repo. Feel free to merge.

File: README.md (hunk 1 above)
Comment (soenkehahn) [reply README.md]: Thanks for the explanation, makes sense.
